utils.js

```diff
@@ -0,0 +1,30 @@
+export default {
+   /**
+    * helper for scrolling in a grid
+    * @function gridScroll
+    * @param {string} id webix id of the grid
+    * @param {int} h horizontal scroll in pixels
+    * @param {int=0} v veritcal scroll in pixels
+    */
+   gridScroll: async (id, h, v = 0) => {
```

Comment:
These would probably better as custom cypress commands so we can reuse them in other tests. (See [`ab_runtime/test/e2e/cypress/support/commands.js`](https://github.com/digi-serve/ab_runtime/blob/master/test/e2e/cypress/support/commands.js)).